nette/notejam/app/presenters/templates/Note/delete.latte

```diff
@@ -0,0 +1,9 @@
+{block title}{$note->name}{/block}
+{block header}{$note->name}{/block}
+{block content}
+	<p>Are you sure you want to delete the note?</p>
+	{form deleteNoteForm}
```

Comment:
Is there a way to not duplicate label texts? I define label while adding the input and when using <label> with `n:` macro.